package.json

```diff
@@ -0,0 +1,113 @@
+    "@react-navigation/elements": "^2.2.5",
+    "@react-navigation/material-top-tabs": "^7.1.0",
+    "@react-navigation/native": "^7.0.14",
+    "@react-navigation/native-stack": "^7.2.0",
+    "@segment/analytics-react-native": "^2.20.3",
+    "@segment/analytics-react-native-plugin-adjust": "^0.7.0",
+    "@segment/analytics-react-native-plugin-clevertap": "^1.1.0",
+    "@segment/analytics-react-native-plugin-destination-filters": "^1.1.0",
+    "@segment/analytics-react-native-plugin-firebase": "^0.4.3",
+    "@segment/sovran-react-native": "^1.1.3",
+    "@walletconnect/react-native-compat": "^2.16.2",
+    "clevertap-react-native": "^2.2.1",
+    "expo": "~52.0.23",
+    "expo-build-properties": "~0.13.0",
+    "expo-constants": "~17.0.4",
+    "expo-dev-client": "~5.0.0",
+    "expo-splash-screen": "~0.29.13",
+    "expo-status-bar": "~2.0.0",
+    "lottie-react-native": "^5.1.6",
+    "react": "18.3.1",
+    "react-native": "0.76.5",
+    "react-native-adjust": "^4.38.1",
+    "react-native-auth0": "^3.2.1",
+    "react-native-camera": "^4.2.1",
+    "react-native-config": "^1.5.3",
+    "react-native-contacts": "https://github.com/valora-inc/react-native-contacts#9940121",
+    "react-native-device-info": "^13.2.0",
+    "react-native-exit-app": "^2.0.0",
+    "react-native-fast-image": "^8.6.3",
+    "react-native-fs": "^2.20.0",
+    "react-native-gesture-handler": "^2.21.2",
+    "react-native-haptic-feedback": "^2.3.3",
+    "react-native-in-app-review": "^4.3.3",
+    "react-native-keychain": "^8.2.0",
+    "react-native-launch-arguments": "^4.0.2",
+    "react-native-linear-gradient": "^2.8.3",
+    "react-native-localize": "^3.3.0",
+    "react-native-pager-view": "^6.4.1",
+    "react-native-permissions": "^4.1.5",
+    "react-native-persona": "^2.2.23",
+    "react-native-quick-crypto": "0.7.7",
+    "react-native-reanimated": "^3.15.1",
+    "react-native-restart": "^0.0.27",
+    "react-native-safe-area-context": "^4.14.1",
+    "react-native-screens": "^4.3.0",
+    "react-native-shake": "5.5.2",
+    "react-native-share": "^11.1.0",
+    "react-native-simple-toast": "^3.3.1",
+    "react-native-sms-retriever": "^1.1.1",
+    "react-native-svg": "^15.7.1",
+    "react-native-video": "^6.6.4",
+    "react-native-webview": "^13.12.5"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.25.2",
+    "@config-plugins/detox": "^9.0.0",
+    "@expo/config-plugins": "~9.0.12",
+    "@faker-js/faker": "^5.5.3",
+    "@mento-protocol/mento-sdk": "^0.2.3",
+    "@react-native-community/cli": "^15.1.3",
+    "@types/fs-extra": "^9.0.13",
+    "@types/react": "~18.3.12",
+    "@walletconnect/sign-client": "^2.13.3",
+    "babel-plugin-module-resolver": "^5.0.2",
+    "detox": "^20.32.0",
+    "dotenv": "^16.3.1",
+    "ethers": "^5.7.2",
+    "expect": "^29.7.0",
+    "lodash": "^4.17.21",
+    "ts-node": "^10.9.2",
+    "ts-retry-promise": "^0.6.0",
+    "twilio": "^3.69.0",
+    "typescript": "^5.3.3",
+    "viem": "^2.21.55"
```

Comment:
Most of these are not needed here. They were just needed for the e2e test in the example app.
We can compare with an expo template to see which are actually needed.

Reply:
oh man i didn't even notice these....yes i was able to remove most of them :D